.github/workflows/test-microk8s.yaml

```diff
@@ -0,0 +1,83 @@
+name: Charmed Katib
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  build:
+    name: Test
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Check out repo
+      uses: actions/checkout@v2
+
+    - name: Install dependencies
+      run: |
+        set -eux
+        sudo snap install microk8s --classic
+        sudo snap install charm --classic
+        sudo snap install juju --classic
+        sudo snap install juju-helpers --classic
+        sudo snap install juju-wait --classic
+        sudo apt update
+
+    - name: Set up dependencies
+      run: |
+        set -eux
+        sudo usermod -a -G microk8s $USER
+        sudo microk8s enable dns storage rbac
+
+    - name: Build Docker images
+      run: |
+        set -eux
+        ./scripts/v1beta1/build.sh -r docker.io/kubeflowkatib -t latest
+        for image in katib-ui katib-controller katib-db-manager; do
```

Comment:
Do you need to update other images also?
For example: `docker.io/kubeflowkatib/file-metrics-collector` ?

Reply:
I actually took out the usage of `./scripts/v1beta/build.sh`, as building all of those images took up too much disk space.

I was thinking about only building the images as they get included in testing, and right now the only test is a simply connectivity test. I'm going to be working on expanding that, but wanted to keep it simple initially.